renderer_elements.js

```diff
@@ -473,6 +541,26 @@ class AccessNode extends Node {
         var textmetrics = ctx.measureText(this.label());
         ctx.fillText(this.label(), this.x - textmetrics.width / 2.0, this.y + LINEHEIGHT / 4.0);
     }
+
+    shade(renderer, ctx, color, alpha='0.6') {
```

Comment:
In all of the `shade` methods below, can't you directly call `this.draw` in the middle?

Reply:
Draw does a lot more usually, including drawing the stroke etc. Only the fill part could be extracted into its own method and reused by both methods. Is that what you mean?